thermo_nerf/field_heads.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class FieldHeadNamesT(Enum):
```

Comment:
I wouldn't move this class in this PR:

1. it is outside the scope of the PR, refactoring this adds more work and more modification for little gains ;). You give yourself more work for no reason
2. FieldHeadNamesT is very dependent on the class it was with in the file so it was nice to have that connection this time. When you put it on the base folder, you remove that. At the very least it should be in the model folder.

All in all I would revert this change